payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -1670,12 +1649,14 @@ class StripeApiRepository @JvmOverloads internal constructor(
         }
 
         // For user key auth, we must create the PM first.
-        val paymentMethodId = requireNotNull(
-            createPaymentMethod(paymentMethodCreateParams, options)?.id
-        )
+        val paymentMethod = createPaymentMethod(
+            paymentMethodCreateParams = paymentMethodCreateParams,
+            options = options,
+        ).getOrThrow()
```

Comment:
Keeping the current behavior. Adding proper error propagation in https://github.com/stripe/stripe-android/pull/7031.